package.json

```diff
@@ -42,6 +42,7 @@
     "css-loader": "^3.1.0",
     "file-loader": "^4.1.0",
     "file-saver": "^2.0.2",
+    "little-loader": "^0.2.0",
```

Comment:
I do not understand why you need this.

You may use dynamic import if you need the runtime. Otherwise you can create a script element and insert it in a few lines of code without extras dependency.

Let me know if it not that simple.

Reply:
Would `import('http://someurl').then(doSomething)` work? I didn't know about it. I will try it.